NDB.Covid19/NDB.Covid19/Utils/MessagingCenterKeys.cs

```diff
@@ -12,5 +12,8 @@ public static class MessagingCenterKeys
         public static string KEY_PERMISSIONS_CHANGED => "PermissionsChangedKey";
 
         public static string KEY_UPDATE_DAILY_NUMBERS => "KEY_UPDATE_DAILY_NUMBERS";
+
+        public static string KEY_APP_BECAME_ACTIVE => "KEY_APP_BECAME_ACTIVE";
```

Comment:
Maybe delete old keys (e.g., KEY_APP_RETURNS_FROM_BACKGROUND and KEY_APP_WILL_ENTER_BACKGROUND if they are no longer used?)

Reply:
The old key is still used in InfectionStatus.
It is unclear to me if we can change the monitored event here to app becomes active/inactive.

I took a look at the implementation in the Danish app, and they use the "old" keys in InfectionStatus.

Reply:
correct :) thanks